packages/main/src/plugin/dashboard/dashboard-service.ts

```
@@ -43,18 +59,116 @@ export class DashboardService {
+    switch (status) {
+      case HEALTH_MONITOR_STATUS.HEALTHY:
+        return 'All systems operational';
+      case HEALTH_MONITOR_STATUS.STABLE:
+        return 'Some systems are stopped';
+      case HEALTH_MONITOR_STATUS.PROGRESSING:
+        // Check if starting or stopping
+        if (allConnections.filter(connection => connection.status === 'starting').length) {
+          return 'Starting up...';
+        } else {
+          return 'Stopping...';
+        }
```

Biome (2.4.6)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)